get-armory-cli

```diff
@@ -1,5 +1,16 @@
 #! /usr/bin/env bash
 
+throw_error_message() {
+   echo "#############################################################################################################"
+   echo "#                           There was a problem trying to download/install avm, and armory"
+   echo "#"
+   echo "# $1"
+   echo "#"
+   echo "# You can keep installing it manually - https://docs.armory.io/cd-as-a-service/setup/cli/#local-installation"
```

Comment:
Unfortunately it doesn't look like there's a way to deep link into the "manual" tab - so it looks like you're just sending them to install with this script again, which would probably be frustrating. I'm also worried that this link will change because the docs team doesn't know there's a dependency here. Can you just send them to the releases page for `avm`? https://github.com/armory/avm/releases